README.md

```diff
@@ -78,7 +78,7 @@ table.select().firstPage().then(result => { ... })
 
 # Where are the tests?
 
-Our tests live in a different repository and tests run on every git push.
+Our tests live in a different repository and tests run on every `git push`.
 We strive to have all of the API covered along with all API error conditions.
 If you hit something that's not right, be sure to ping us on intercom or open
-a github issue.
+a Github Issue.
```

Comment:
"Github" should be "GitHub" (with a capital H).

Reply:
Nice catch.  Will fix shortly.